client/components/Omnichannel/modals/CloseChatModal.js

```diff
@@ -46,6 +47,19 @@ const CloseChatModal = ({ department = {}, onCancel, onConfirm }) => {
 		);
 	}, [department, tagRequired, t, tags]);
 
+	if (!commentRequired && !tagRequired) {
```

Comment:
`tagRequired` is always returning **false**, but I have defined **true** on the department.